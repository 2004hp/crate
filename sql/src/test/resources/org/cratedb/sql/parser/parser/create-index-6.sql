create index myIndex on myTable using fulltext(myTable.myColumn) with (analyzer='myanalyzer')