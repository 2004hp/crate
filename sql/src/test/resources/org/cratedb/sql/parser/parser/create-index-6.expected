org.cratedb.sql.parser.parser.CreateIndexNode@51827002
name: i1
statementType: CREATE INDEX
unique: false
indexName: i1
tableName: t1
indexMethod: fulltext
indexColumns: 
indexProperties: {analyzer=value: german
type: CHAR(6) NOT NULL
}
existenceCheck: NO_CONDITION
	
	org.cratedb.sql.parser.parser.IndexColumnList@4a87bd1c

	[0]:	
	org.cratedb.sql.parser.parser.IndexColumn@15909f7e
	columnName: c1
	tableName: null
	ascending
	[1]:	
	org.cratedb.sql.parser.parser.IndexColumn@49d71441
	columnName: c2
	tableName: null
	ascending
